pkg/internal/runtime/runtime.go

```diff
@@ -18,6 +18,8 @@ func GetDefault(logger log.Logger) cluster.ProviderOption {
 	case "docker":
 		logger.Warn("using docker due to KIND_EXPERIMENTAL_PROVIDER")
 		return cluster.ProviderWithDocker()
+	case "nerdctl", "finch", "nerdctl.lima":
```

Comment:
`logger.Warn("using nerdctl due to KIND_EXPERIMENTAL_PROVIDER")`

Reply:
Good catch; repushed. Thanks!